### DISCUSSION TITLE
Auth: localStorage / Jamstack

### DISCUSSION THREAD

**Discussion by charlie17** [supabase/supabase#1261] [Questions]
Been reading all over (I am no security expert) that localStorage should never be used to store auth tokens. Yet it seems that is what Supabase/GoTrue are doing by default? Is this the recommended approach for a Supabase Jamstack setup? (i.e. assume no server, but access to Lambda functions).

Any alternative approaches? Did not see much on this in the docs?

Thanks

**Comment by chipilov** [comment]
It seems that a very similar question was asked in another thread and got a response from the Supabase team (although, imho, the answer is NOT definitive):

https://github.com/supabase/supabase/discussions/1663
